src/transformers/models/data2vec/configuration_data2vec_vision.py

```diff
@@ -40,9 +40,6 @@ class Data2VecVisionConfig(PretrainedConfig):
     [facebook/data2vec-vision-base](https://huggingface.co/facebook/data2vec-vision-base) architecture.
 
     Args:
-        vocab_size (`int`, *optional*, defaults to 8092):
```

Comment:
I can't find any use of this, despite the docstring below was changed to say `the number of different image tokens that can be used
            during pre-training.`

Reply:
This is because Data2VecVision was copied from BEiT. BEiT has a `BeitForMaskedImageModeling` class which uses this attribute. But for Data2Vec it's not required since the plan was to add a `Data2VecVisionForPreTraining` class at some point